docs/index.md

```diff
@@ -0,0 +1,109 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The In-App payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
+
+You need to have a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). The Payments SDK will allow you to approve the order on the client side of your app by using the buyer's card information.
+
+## Eligibility
+```
+# Placeholder: Product Eligibility
+what kind of merchants are we targetting?
+what features do we offer right now/what features are we missing?
+```
+
+## Integration methods
+
+- [Card Fields UI Integration](#card-fields-ui-integration)
+- [Fully Customizable Integration](#fully-customizable-integration)
+
+## Card Fields UI Integration
+
+Accept card payments using our card fields UI
+
+```
+# Placeholder: Include images of card fields
+```
+
+**Requirements:**
+Card fields UI offers a low effort integration where you can use our card fields UI, and our card fields will handle ...
+
+## Fully Customizable Integration
+
+Process card payments with PayPal Payments SDK using your own UI.
+
+### Steps
+
+#### 1. Create an order
+Call `v2/checkout/orders` to create an order an obtain an order ID
+
+```
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <access_token>' \
```

Comment:
Should we include docs here for how to obtain the access token or will those live somewhere else?

Reply:
These felt like placeholders I'm not sure where that would live. Honestly, a lot of this will be duplicated across web, iOS and Android repos.

Reply:
I think we could provide links to the existing documentation for getting an access token: https://developer.paypal.com/api/rest/#link-getanaccesstoken

A link to the create order documentation may help as well: https://developer.paypal.com/api/orders/v2/#orders_createe

Reply:
In the future we could link to the server SDKs. I'm hesitant to add links at the moment since these are placeholders. `curl` provides a good platform-agnostic way to illustrate steps to fetch a client token.

Reply:
@tdchow I'll reference the links though for good measure that's a good call.